### PR TITLE
upgrade sbc-common-component version, add its depending env configs

### PR DESCRIPTION
**Pull request by garyfan1**
*Issue #:* /bcgov/entity#12814

*Description of changes:*
upgrade sbc-common-component version, add its depending env configs

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/utils/config-helper.ts [hunk 1]
```diff
@@ -136,4 +136,9 @@ export async function fetchConfig (): Promise<any> {
     (<any>window).addressCompleteKey = addressCompleteKey
     console.info('Set Address Complete Key.')
   }
+
+  const siteminderLogoutUrl: string = response.data.SITEMINDER_LOGOUT_URL
+  if (siteminderLogoutUrl) {
+    sessionStorage.setItem('SITEMINDER_LOGOUT_URL', siteminderLogoutUrl)
```
Comment (dimak1) [inline ppr-ui/src/utils/config-helper.ts:142]: Is this used anywhere?

Comment (cameron-eyds) [reply ppr-ui/src/utils/config-helper.ts:142]: I was curious about this as well. 
Gary mentioned this was intentional as it's a dependency of recent changes in the SBC Common Components, although i'm not sure the details.

**Review by cameron-eyds — APPROVED** [review]
@garyfan1 Code looks good!

Could you call out the purpose of the siteminder inclusion in both the ticket/pr.
What it does, why we need etc, just so everyone is aware, thanks sir!

**Comment by garyfan1** [comment]
We weren't logging out of siteminder when logging out, and this will cause an error if we try to log in again with another IDIR.
More details please see https://github.com/bcgov/entity/issues/12565.

**Comment by cameron-eyds** [comment]
> We weren't logging out of siteminder when logging out, and this will cause an error if we try to log in again with another IDIR. More details please see [bcgov/entity#12565](https://github.com/bcgov/entity/issues/12565).

Excellent, thanks 👍

**Review by dimak1 — APPROVED** [review]
Thanks for clarification @garyfan1. Approved.

**Pull request merged**